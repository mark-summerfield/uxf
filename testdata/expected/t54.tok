FileComment Str("File-level comment")
TClassBegin ttype=<Pair>
  Field Str("first") vtype=str
  Field Str("second") vtype=str
TClassEnd
MapBegin
MapEnd
Eof
----------------------------------------
