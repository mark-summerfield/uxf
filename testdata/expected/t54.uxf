uxf 1.0
#<File-level comment>
=Pair first:str second:str
{}
