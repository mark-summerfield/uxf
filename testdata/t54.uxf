uxf 1
#<File-level comment>
= Pair first:str second:str
{}
