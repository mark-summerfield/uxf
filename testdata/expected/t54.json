{
  "UXF^comment": "File-level comment",
  "UXF^ttypes": [
    {
      "UXF^ttype": {
        "name": "Pair",
        "fields": {
          "first": "str",
          "second": "str"
        }
      }
    }
  ],
  "UXF^data": {}
}